examples/lstm_ptb/README.md

```diff
@@ -25,55 +25,68 @@ Let's take a look at signature file:
   "inputs": [
     {
       "data_name": "data",
-      "data_shape": [1, 60]
-    }
-  ],
-  "input_type": "application/json",
-  "outputs": [
-    {
-      "data_name": "softmax",
-      "data_shape": [1, 10000]
+      "data_shape": [
+        1,
+        60
+      ]
     }
-  ],
-  "output_type": "application/json"
+  ]
 }
 ```
-Both input and output are of type application/json. Input data shape is (1, 60). For sequence to sequence models, the inputs can be variable length sequences. In the signature file the input shape should be set to the maximum length of the input sequence, which is the default bucket key. The bucket sizes are defined when training the model. In this example valid bucket sizes are 10, 20, 30, 40, 50 and 60. Default bucket key is the maximum value which is 60. Check [training details](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/cudnn_lstm_bucketing.py) if you want to know more about the bucketing module in MXNet. Output shape is (1, 10000), since PTB data set contains 10,000 vocabularies.
+Input data shape is (1, 60). For sequence to sequence models, the inputs can be variable length sequences. In the signature file the input shape should be set to the maximum length of the input sequence, which is the default bucket key. The bucket sizes are defined when training the model. In this example valid bucket sizes are 10, 20, 30, 40, 50 and 60. Default bucket key is the maximum value which is 60. Check [training details](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/cudnn_lstm_bucketing.py) if you want to know more about the bucketing module in MXNet. Output shape is (1, 10000), since PTB data set contains 10,000 vocabularies.
```

Comment:
rephrase the last part of the paragraph as : (beginning from since ... ) 
since the Penn Tree Bank dataset has a vocabulary size of 10,000.

Reply:
I didn't change this part.
We will have 2nd round of scrub about wording with Aaron next week.